boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/JDBCBoosterPackConfigurator.java

```diff
@@ -56,4 +58,10 @@ public void writeConfigToServerXML(Document doc) {
         lib.appendChild(fileLoc);
         serverRoot.appendChild(lib);
     }
+
+	@Override
+	public void setFeatureString(String feature) {
+		featureGAV = feature;
```

Comment:
Shouldn't getFeatureString() return featureGAV?

Reply:
i conceived of this as a config generator that might render different config for different levels of the technology. not saying this is the right thing to do, but made it work that way - so once we know what level of technology we are dealing with via the dependency resolution, i am setting the specific value here so when the server.xml config step occurs this code will have a control point to write out the right config. having said that i am not utilizing a getter for that string ... so maybe i should use another approach for getting that technology string passed in.